assets/js/blocks/product-category/block.js

```diff
@@ -138,9 +174,20 @@ class ProductByCategoryBlock extends Component {
 	}
 
 	renderEditMode() {
-		const { attributes, debouncedSpeak, setAttributes } = this.props;
+		const { attributes, debouncedSpeak } = this.props;
+		const { changedAttributes } = this.state;
+		const currentAttributes = { ...attributes, ...changedAttributes };
```

Comment:
This is where I merge the unsaved attributes (from state) with the saved attributes (from this.props). I think I'm doing this correctly?

Reply:
I think so too. :slightly_smiling_face: